maintainers/maintainer-list.nix

```diff
@@ -1279,6 +1279,12 @@
     githubId = 37193992;
     name = "Arthur Teisseire";
   };
+  arti5an = {
+    email = "artis4n@outlook.com";
+    github = "arti5an";
+    githubId = 14922630;
+    name = "Richard Smith";
+  };
```

Comment:
Add the maintainer in a separate commit.

Reply:
The format can refer to https://github.com/NixOS/nixpkgs/pull/222132/commits

Reply:
I've split the commits as requested. Makes sense as I've since seen that pattern throughout the commit log.